### PR TITLE
301/small updates

### PR DESCRIPTION
**Pull request by ebrett**
### Context

as stated, some small updates

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -178,7 +178,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 6..128
+  config.password_length = 8..128
```
Comment (guy-roberts) [inline config/initializers/devise.rb:182]: The short minimum length does worry me, but even I am tired of hearing my own voice !

File: db/seeds/create_users.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    u.role = "editor"
+    u.password = "11abXY!@"
+    u.role = "admin"
   end
 
   User.create! do |u|
     u.email = "guy@home.com"
-    u.password = "guyguy"
+    u.password = "11abXY!@"
     u.role = "editor"
```
Comment (guy-roberts) [inline db/seeds/create_users.rb:11]: We need to avoid seeding public servers with passwords that are in the repo

Comment (ebrett) [reply db/seeds/create_users.rb:11]: agreed, but what's the alternative?

File: config/initializers/devise.rb (hunk 1 above)
Comment (ebrett) [reply config/initializers/devise.rb:182]: yea, they've warn me down as well

**Pull request closed without merging**